pkgs/applications/virtualization/runc/default.nix

```
@@ -35,7 +35,7 @@ buildGoModule rec {
   buildPhase = ''
     runHook preBuild
     patchShebangs .
-    make ${toString makeFlags} runc man
+    make ${toString makeFlags} runc man SHELL=$(command -v bash)
```

Maybe it would be better this way?

```suggestion
    make ${toString makeFlags} runc man SHELL=${stdenv.shell}
```

---

The suggested modification is wrong. `stdenv.shell` will be picked from `hostPlatform` instead of `buildPlatform` as it's not altered in scope (because it's not in `nativeBuildInputs`, `depsBuildBuild`, etc), thus breaking cross compilation. See https://nixos.org/manual/nixpkgs/unstable/#chap-cross

---

No?

```nix
nix-repl> :a import <nixpkgs> { crossSystem = "riscv64-linux"; }
Added 22836 variables.

nix-repl> stdenv.shell
"/nix/store/8jind2ncd72hlzym9pghjjdj79fvqzv5-bash-5.2p32/bin/bash"

nix-repl> runtimeShell
"/nix/store/b25fb94qjqh7k5q5naiyahk274c1s054-bash-riscv64-unknown-linux-gnu-5.2p32/bin/bash"

nix-repl>
```

---

My mistake, sorry.